aspnetcore/fundamentals/servers/index.md

```diff
@@ -32,6 +32,26 @@ When using [IIS](/iis/get-started/introduction-to-iis/introduction-to-iis-archit
 
 The [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module) is a native IIS module that handles native IIS requests between IIS and the in-process IIS HTTP Server or Kestrel. For more information, see <xref:host-and-deploy/aspnet-core-module>.
 
+<a name="korh"></a>
+
+## Kestrel vs. HTTP.sys
+
+Kestrel has the following advantages over HTTP.sys:
+
+  * Better performance and memory utilization.
+  * Cross platform
+  * Agility, it's developed and patched independent of the OS.
+  * Programmatic port and TLS configuration
+  * Extensibility that allows for protocols like [PPv2](https://github.com/aspnet/AspLabs/blob/master/src/ProxyProtocol/ProxyProtocol.Sample/ProxyProtocol.cs) and alternate transports.
+
+Http.Sys operates as a shared kernel mode component with the following features that kestrel does not have:
+
+  * Port sharing
+  * Kernel mode windows authentication. [Kestrel supports only user-mode authentication](xref:security/authentication/windowsauth#kestrel).
+  * Caching
+  * Fast proxying via queue transfers
+  <!-- * Potentially faster file transfers. -->
```

Comment:
I'll remove this comment and change to ***faster file transfers.*** after we have benchmarks.